### PR TITLE
Add error mapping

### PR DESCRIPTION
**Pull request by dlehren**
Added error mapper with list of error codes to show in cardknox_error_mapping.xml.
Moved the message box lower in the form for users to see it.
Removed extra JS logging.
Bumped to version 1.0.12

### CODE REVIEW

**Comment by dlehren** [comment]
@mstein-ck please have a look when you have a chance

File: view/frontend/web/template/payment/cardknox-form.html [hunk 1]
```diff
@@ -115,6 +112,10 @@
             </fieldset>
         </form>
 
+        <!-- ko foreach: getRegion('messages') -->
+        <!-- ko template: getTemplate() --><!-- /ko -->
+        <!--/ko-->
+
```
Comment (mstein-ck) [inline view/frontend/web/template/payment/cardknox-form.html:118]: I don't see where you moved the message box down on the page, is this it?

Comment (dlehren) [reply view/frontend/web/template/payment/cardknox-form.html:118]: Yes.

File: Gateway/Request/AuthorizationRequest.php [hunk 2]
```diff
@@ -43,7 +43,8 @@ public function build(array $buildSubject)
             'xCurrency' => $order->getCurrencyCode(),
             'xCardNum' => $payment->getAdditionalInformation("xCardNum"),
             // always true; order number is incremented on every attempt so invoice is always different
-            'xIgnoreInvoice' => true
+            'xIgnoreInvoice' => true,
+            'xTimeoutSeconds' => 55
```
Comment (dlehren) [inline Gateway/Request/AuthorizationRequest.php:47]: It seems like an integer is good. I tested and it worked.

**Review by mstein-ck — APPROVED** [review]

**Pull request merged**